src/App.js

```diff
@@ -1,22 +1,24 @@
-import React, { Component } from 'react';
```

Comment:
If you `import react`, then there's no need to import individual components. You've already imported all of react. Just use `import * as React from 'react` and you'll be able to access anything in the namespace. For example, `Component` just becomes `React.Component`. 

If you just want the shorthand though then leaving `import { Component } from 'react'` is fine, At that point though you don't really need the `import React from 'react` part.